### PR TITLE
RGR-688 Error messages within the ops chat on PS tablet not consistent with ACs

### PR DESCRIPTION
**Pull request by elonmitchell**
Problem
=======
- Error messages within the ops chat on PS tablet not consistent with ACs

[link to JIRA RGR-688](https://usxtech.atlassian.net/browse/RGR-688)

Solution
========
- Implemented the Freshchat init status

Screenshots (optional):
-----------------------
Screenshot of Android device

### CODE REVIEW

File: src/components/VariantChat.tsx [hunk 1]
```diff
@@ -3,43 +3,53 @@ import { StyleSheet, Text, View } from 'react-native';
...
+    // Success
     return <Chat {...props} />;
+  } else if (initStatus === FreshchatInit.Fail) {
+    // Failed
+    if (NoConversationComponent) {
+      return NoConversationComponent;
+    } else if (initErrorMessage) {
+      return (
+        <View style={styles.container}>
+          <Text style={styles.textNoCoversation}>{initErrorMessage}</Text>
```
Comment (ajp8164) [inline src/components/VariantChat.tsx:31]: Spelling typo

File: src/components/VariantChat.tsx [hunk 2]
```diff
@@ -3,43 +3,53 @@ import { StyleSheet, Text, View } from 'react-native';
...
-    return NoConversationComponent;
-  } else {
-    return (
-      <View style={styles.container}>
-        <Text>{`Conversation '${channelName}' could not be loaded.`}</Text>
-      </View>
-    );
-  }
+  // Loading...
+  return <View style={styles.container} />;
```
Comment (ajp8164) [inline src/components/VariantChat.tsx:38]: This is just an empty view? Doesn’t this freeze the app with no way out?

Comment (elonmitchell) [reply src/components/VariantChat.tsx:38]: yes, it's just an empty view.

Comment (ajp8164) [reply src/components/VariantChat.tsx:38]: Oh. I don’t think this line will ever execute. Will fall into one of the returns in the above if-then-else.

**Review by ajp8164 — APPROVED** [review]
ACK

**Pull request merged**